_posts/de/newsletters/2022-10-26-newsletter.md

```diff
@@ -0,0 +1,357 @@
+  öffentlichen Diskussion zu beteiligen.
+
+## Nennenswerte Code- und Dokumentationsänderungen
+
+*Erwähnenswerte Änderungen diese Woche in [Bitcoin Core][bitcoin core repo],
+[Core Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], und [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #23443][] fügt eine neue P2P-Protokollnachricht, `sendtxrcncl`
+  (send transaction reconciliation), hinzu, die es einem Knoten erlaubt, einem
+  Peer zu signalisieren, dass er [erlay][topic erlay] unterstützt. Dieser PR
+  fügt nur den ersten Teil des erlay-Protokolls hinzu - andere Teile werden
+  benötigt, bevor es verwendet werden kann.
+
+- [Eclair #2463][] und [#2461][eclair #2461] aktualisieren die Eclair-
+  Implementierung der [interaktiven und dualen Finanzierungsprotokolle][topic
+  dual funding], welche erfordert, dass sich jeder Finanzierungsinput für
+  [RBF][topic rbf] anmeldet und auch bestätigt wird (d.h. einen Output ausgibt,
+  der sich bereits in der Blockchain befindet). Diese Änderungen stellen sicher,
+  dass RBF verwendet werden kann und dass keine der Gebühren, die ein
+  Eclair-Benutzer beisteuert, dazu verwendet wird, frühere Transaktionen seiner
+  Peers zu bestätigen.
+
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="23443,2463,2461,25353" %}
+[bitcoin core 24.0 rc2]: https://bitcoincore.org/bin/bitcoin-core-24.0/
+[ldk 0.0.112]: https://github.com/lightningdevkit/rust-lightning/releases/tag/v0.0.112
+[bcc testing]: https://github.com/bitcoin-core/bitcoin-devwiki/wiki/24.0-Release-Candidate-Testing-Guide
+[news222 rbf]: /en/newsletters/2022/10/19/#transaction-replacement-option
+[kotliar free option]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-October/021056.html
+[sanders cpfp]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-October/021060.html
+[rubin cpfp]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-October/021059.html
+[riard free option]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-October/021067.html
+[zhao no control]: https://www.erisian.com.au/bitcoin-core-dev/log-2022-10-20.html#l-440
+[towns uncoordinated]: https://www.erisian.com.au/bitcoin-core-dev/log-2022-10-20.html#l-488
+[sanders doubt]: https://www.erisian.com.au/bitcoin-core-dev/log-2022-10-20.html#l-490
+[towns uasf]: https://www.erisian.com.au/bitcoin-core-dev/log-2022-10-20.html#l-492
+[zumsande option]: https://www.erisian.com.au/bitcoin-core-dev/log-2022-10-20.html#l-493
+[coredev xs]: https://diyhpl.us/wiki/transcripts/bitcoin-core-dev-tech/
+[p2p encryption]: https://diyhpl.us/wiki/transcripts/bitcoin-core-dev-tech/2022-10-10-p2p-encryption/
+[news222 bip324]: /en/newsletters/2022/10/19/#bip324-update
```

Comment:
I will change the link to the DE newsletter, once de-222 has been merged in.

Reply:
@tschweg sounds good. Do you have a 222 in the works? /shall we wait to merge this 223 for 222?

Reply:
@bitschmidty 222 is in the works, but it'll probably take me until Tuesday EOB UTC to finish it. The idea was to push out 223 first since it's the current issue, but happy to wait until 222 is in.

Reply:
@tschweg sounds good, merged 223!